packages/core/src/common/keybinding.ts

```diff
@@ -187,11 +286,21 @@ export class KeybindingRegistry {
         if (event.defaultPrevented) {
             return;
         }
-        const keyCode = KeyCode.createKeyCode(event);
+        const keyCode = TheiaKeyCodeUtils.createKeyCode(event);
         const binding = this.getKeybindingForKeyCode(keyCode);
         if (!binding) {
             return;
         }
+        const bindingsForThisCommand = this.getKeybindingsForCommand(binding.commandId);
```

Comment:
It is not clear to me, why you do these checks here. We already have a binding, couldn't we just execute it?